web/src/main/java/se/inera/intyg/webcert/web/service/patient/PatientDetailsResolverImpl.java

```diff
@@ -248,6 +249,22 @@ private Patient resolvePatient(Personnummer personnummer, WebCertUser user, Pati
         return patient;
     }
 
+    private Patient resolvePatient(Personnummer personnummer, PatientDetailResolveOrder resolveOrder) {
```

Comment:
resolveOrder används inte.

Reply:
Fixat. Checkar in ändringar samtidigt när du är klar med reviewn.